lib/basevm.py

```diff
@@ -15,29 +15,13 @@ def __init__(self, uuid, host):
         self.host = host
 
     def param_get(self, param_name, key=None, accept_unknown_key=False):
-        return _param_get(self.host, self.xe_prefix, self.uuid, param_name, key, accept_unknown_key)
+        return _param_get(self.host, BaseVM.xe_prefix, self.uuid, param_name, key, accept_unknown_key)
```

Comment:
What's the benefit in hardcoding the class name here and in other places ?  It is likely to cause unwanted issues or at least extra work on class rename, copy-paste, etc